app/api/groups/update_permissions.go

```diff
@@ -43,6 +44,19 @@ type updatePermissionsInput struct {
 	// The current user should have permissions_generated.can_grant_view_generated >= 'enter' in order to
 	// increase this field's value.
 	CanEnterUntil time.Time `json:"can_enter_until" validate:"can_enter_until"`
+	// Optional
+	// The current user should have `permissions_generated.can_grant_view` >= 'content',
+	// in order to set this field's value.
+	CanRequestHelpTo setCanRequestHelpTo `json:"can_request_help_to" validate:"can_request_help_to_set,can_request_help_to_consistent,can_request_help_to_can_grant_view_content,can_request_help_to_visible"` //nolint:lll
+}
+
+// Cannot set `id` and `is_all_users_group` at the same time.
+type setCanRequestHelpTo struct {
+	// The given group must be visible by both the current user and `group_id`.
+	// Can be set to `null` to set the helper group to no group.
+	ID *int64 `json:"id" sql:"column:can_request_help_to"`
```

Comment:
So I assume it should be marked as`Optional` as well.

I assume it should be marked as `nullable` (I don't remember if it is an "official" tag but at least it is how we did so far)

Reply:
fixed